.github/workflows/release.yml

```diff
@@ -0,0 +1,102 @@
+---
+
+name: Release
+
+on:
+  pull_request:
+    branches: main
+    types: closed
+
+jobs:
+  publish_platform:
+    name: Publish Platform
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Clean
+        uses: easimon/maximize-build-space@v4
+        with:
+          root-reserve-mb: 25600
+          swap-size-mb: 1024
+          remove-dotnet: 'true'
+
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup
+        run: |
+          sudo apt update
+          sudo apt install -y ccache flatpak flatpak-builder gir1.2-ostree-1.0 python3-aiohttp python3-gi python3-tenacity xvfb zstd
+          sudo rm -rf /var/lib/{apt,dpkg,cache,log}/
+
+          curl https://raw.githubusercontent.com/flatpak/flat-manager/0.3.7/flat-manager-client > ./flat-manager-client
+          sudo mv ./flat-manager-client /usr/bin/flat-manager-client
+          sudo chown root: /usr/bin/flat-manager-client
+          sudo chmod +x /usr/bin/flat-manager-client
+
+          sudo flatpak remote-add --system --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo
+
+      - name: Cache
+        uses: actions/cache@v2
+        with:
+          path: ./.flatpak-builder
+          key: flatpak-builder-${{ github.sha }}
+          restore-keys: |
+            flatpak-builder-
+            flatpak-
+
+      - name: Build
+        env:
+          DISPLAY: "0:0"
+        run: |
+          sudo xvfb-run --auto-servernum flatpak-builder --disable-rofiles-fuse --keep-build-dirs --install-deps-from=flathub --ccache --repo=elementary builddir ./io.elementary.Sdk.json
```

Comment:
`flatpak-builder` accepts a `--default-branch=BRANCH` arg. So this should include `--default-branch=6.0` or even better, something dynamic like `--default-branch=${GITHUB_REF#refs/tags/}`.

Reply:
For apps, yes. For the platform, no. It should be a set version so apps can target a specific platform and not worry about breaking changes when we release 6.1 or 7.0. See how we target the gnome platform here:

```js
{
    "runtime-version": "3.38",
    "runtime": "org.gnome.Platform",
}
```

Apps:

```
Name                                                    Application ID                                          Version              Branch             Origin                             Installation
Discord                                                 com.discordapp.Discord                                                       stable             flathub                            user
Postman                                                 com.getpostman.Postman                                                       stable             flathub                            user
sequeler                                                com.github.alecaddd.sequeler                                                 stable             flathub                            user
fondo                                                   com.github.calo001.fondo                                                     stable             flathub                            user
Slack                                                   com.slack.Slack                                                              stable             flathub                            user
Client                                                  com.spotify.Client                                                           stable             flathub                            user
DBeaverCommunity                                        io.dbeaver.DBeaverCommunity                                                  stable             flathub                            user
```

Platforms:

```
Name                                                    Application ID                                          Version              Branch             Origin                             Installation
GNOME Application Platform version 3.28                 org.gnome.Platform                                                           3.28               flathub                            system
Platform                                                org.gnome.Platform                                                           3.32               flathub                            user
GNOME Application Platform version 3.34                 org.gnome.Platform                                                           3.34               flathub                            system
Platform                                                org.gnome.Platform                                                           3.34               flathub                            user
Platform                                                org.gnome.Platform                                                           3.36               flathub                            user
GNOME Application Platform version 3.38                 org.gnome.Platform                                                           3.38               flathub                            system
Platform                                                org.gnome.Platform                                                           3.38               flathub                            user
GNOME Application Platform version 40                   org.gnome.Platform                                                           40                 flathub                            system
GNOME Application Platform version 40                   org.gnome.Platform                                                           40                 flathub                            user
```

Reply:
Alright that makes sense, thanks for the explanation!